content/articles/rust-for-web-services.md

```diff
@@ -0,0 +1,132 @@
+Title: Trying Rust for web services
+Date: 2015-09-30
+Short_summary: I made a very basic Rust backend, here's what I think about it
+Category: Dev
+Authors: Vincent
+
+*Reading time: ~15 minutes.*
+
+
+Developing web apps in a dynamic language  is a breeze when using frameworks like [Django](https//www.djangoproject.com/) for Python.
```

Comment:
Developing web apps in a dynamic languages

(keep the singular or plural case consistent)